xpdacq/xpdacq.py

```diff
@@ -390,26 +395,31 @@ def __call__(self, sample, plan, subs=None, *,
                   "beamtime object, scan will keep going....")
         metadata_kw.update(sample)
         sh = glbl.shutter
-        # force to open shutter before scan and close it after
-        if glbl.shutter_control:
-            plan = bp.pchain(bp.abs_set(sh, 1), plan, bp.abs_set(sh, 0))
+
         # Alter the plan to incorporate dark frames.
         if glbl.auto_dark:
             plan = dark_strategy(plan)
             plan = bp.msg_mutator(plan, _inject_qualified_dark_frame_uid)
+
+        # force to open shutter before scan and close it after
+        if glbl.shutter_control:
```

Comment:
I am worried that another number could mean an error or something else that is neither open nor closed.